### DISCUSSION TITLE
Welcome to Fractal Designer Discussions!

### DISCUSSION THREAD

**Discussion by Teddy-van-Jerry** [Teddy-van-Jerry/Fractal_Designer#15] [Announcements]
## 👋 Welcome to FRD Community!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Visit Fractal Designer website at https://frd.teddy-van-jerry.org.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by L-V-Beethoven** [comment]
Hello, I am L.V.Beethoven and new to this community! I really want to add more music features to the software.

**Reply by Teddy-van-Jerry** [reply]
Glad to hear that! We all want to contribute to better the software!
